pkg/kubeadm/master/manifests.go

```diff
@@ -187,10 +237,9 @@ func componentPod(container api.Container, volumes ...api.Volume) api.Pod {
 }
 
 func getComponentCommand(component string, s *kubeadmapi.KubeadmConfig) (command []string) {
-	baseFalgs := map[string][]string{
+	baseFlags := map[string][]string{
 		apiServer: []string{
-			"--address=127.0.0.1",
-			"--etcd-servers=http://127.0.0.1:2379",
```

Comment:
So here are a couple changes.

1. `--address` is deprecated in favor of `--insecure-bind-address`.
2. `--etcd-servers` must be set to a different value, depending on if the user specified `--external-etcd-endpoints`.